wiki/Daily_challenge/en.md

```diff
@@ -0,0 +1,65 @@
+# Daily Challenge
+
+The **daily challenge** is a multiplayer mode in osu!(lazer) where players are able to build up a *daily* streak by passing beatmaps consecutively with increasing difficulty.
```

Comment:
```suggestion
The **daily challenge** is a multiplayer mode in osu!(lazer) where players are able to build up a *daily* streak by passing one predetermined beatmap each day.
```
The increase in difficulty in mentioned in line 5 so I think it's better to avoid a repetition.
Only stating "by passing beatmaps consecutively" one might be mislead into believing that it's a marathon where you need to play multiple maps in a row. The proposed change should clarify that it's one map per day for the streak

Reply:
I would rather remove the repetition in the line below. Keep in mind, the most important information should come first.

The text mentioning that the difficulty resets each 7 days *may* be moved to the first paragraph as well.

Reply:
Went with Cookie's suggestion for now, will leave open for discussion.